cities/crud.py

```diff
@@ -0,0 +1,55 @@
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from cities import models, schemas
+
+
+async def create_city(
+    db: AsyncSession,
+    city: schemas.CityCreate
+) -> models.City:
+    db_city = models.City(name=city.name, additional_info=city.additional_info)
+    db.add(db_city)
+    await db.commit()
+    await db.refresh(db_city)
+    return db_city
+
+
+async def get_cities(db: AsyncSession) -> list[models.City]:
+    result = await db.execute(select(models.City))
+    return result.scalars().all()
+
+
+async def get_city(db: AsyncSession, city_id: int) -> models.City | None:
+    result = await db.execute(
+        select(models.City).filter(models.City.id == city_id)
+    )
+    return result.scalar_one_or_none()
+
+
+async def update_city(
+    db: AsyncSession,
+    city_id: int,
+    city: schemas.CityCreate
+) -> models.City | None:
```

Comment:
The return type hint `models.City | None` requires Python 3.10 or later. If you're using an earlier version, consider using `Optional[models.City]` from the `typing` module.